Binary_Search/Binary_Search.py

```diff
@@ -1,33 +1,43 @@
 # Function for Binary Search
-def Binary_Search(list, desired):
+def binarySearch(array, desired):
```

Comment:
Use this for naming convention in python https://docs.quantifiedcode.com/python-anti-patterns/readability/using_camelcase_in_function_names.html